installer/win/WixSharpInstaller/Symphony.cs

```diff
@@ -258,11 +273,68 @@ public static ActionResult UpdateConfig(Session session)
     // the requested replacement performed.
     static string ReplaceProperty( string data, string name, string value )
     {
-        // Using regular expressions to replace the existing value in the config file with the 
+        // Using regular expressions to replace the existing value in the config file with the
         // one from the property. This is the same as the regex we used to have in the old
         // Advanced Installer, which looked like this: "url"\s*:\s*".*" => "url": "[POD_URL]"
         return System.Text.RegularExpressions.Regex.Replace(data, @"""" + name + @"""\s*:\s*"".*""",
             @"""" + name + @""":""" + value.Trim() + @"""");
     }
 
+    // CleanRegistry custom action
+    [CustomAction]
+    public static ActionResult CleanRegistry(Session session)
+    {
+        try
+        {
+            // Remove registry keys added for auto-launch
+
+            using( var key = Registry.CurrentUser.OpenSubKey(@"Software\Microsoft\Windows\CurrentVersion\Run", true) )
+            {
+                if (key != null)
+                {
+                    key.DeleteValue("Symphony", false);
```

Comment:
what is false in this case? Can we move it to a variable and use the variable here so the code is self explanatory?

Same for all the following occurrences.

Reply:
it is just the API for C# DeleteValue - second parameter is throwOnMissingKey or something like that. But I don't think we need a variable for this, I think it is a case where you would look up the API function anyway, as it is a Win32 API function and there are usually subtleties there if you want to understand what is happening